src/js/components/form-elements/SelectField.jsx

```diff
@@ -91,10 +11,7 @@ const SelectField = (props) => {
       {...attributes}
       className={`select-xs ${className}`}
       classNamePrefix="react-select"
-      customSelectComponents={{
-        Menu,
-        Option,
-      }}
+      customSelectComponents={{}}
```

Comment:
`customSelectComponents={{}}` empty object prop seems unnecessary since you already have defined 

```
Select.defaultProp = {
    customSelectComponents: {}
}
```

Reply:
Yup, you're right, fixed